vic/drivers/cesm/bld/vic.globalconfig.txt

```diff
@@ -77,3 +87,11 @@ OUTVAR      OUT_SWE
 OUTVAR      OUT_SOIL_MOIST
 OUTVAR      OUT_ALBEDO
 OUTVAR      OUT_SOIL_TEMP
+OUTVAR      OUT_R_NET
+OUTVAR      OUT_LATENT
+OUTVAR      OUT_SENSIBLE
+OUTVAR      OUT_RAD_TEMP
+OUTVAR      OUT_SURF_TEMP
+OUTVAR      OUT_SWNET
+OUTVAR      OUT_LWNET
+OUTVAR      OUT_GRND_FLUX
```

Comment:
Having something that let's us identify active layer depth would be useful, but now sure whether that would be part of every run. Same for all these met variables. Can we split out the radiation variables (down and up rather than just net).

Reply:
That's what I was thinking too. From knowing how much outputting data for 10 thermal nodes increases the size of the output files, I think we should add that when we need it and not make it the default. I'll split out the radiation variables and remove some of the met ones

Reply:
Remember this is just a monthly output file so the cost of writing out 3d vars is much less than it would otherwise be. My thought is that you should write out more variables, unless you have reason to think the data volumes will be out of control or that it is negatively impacting runtimes.

Reply:
I added soil node temperature to the monthly output files as well as eliminating some of the met vars and splitting out the radiation vars. @bartnijssen and @jhamman - what do you think of the updated ones?